src/create.cpp

```diff
@@ -149,6 +151,9 @@ namespace create {
       }
     } // if CREATE_2
 
+    totalLeftDist += leftWheelDist;
+    totalRightDist += rightWheelDist;
```

Comment:
Since these are floats, the value wouldn't wrap around, it would just become +/- infinity. The bigger issue here, though is loss of precision. I did some quick tests and it looks like the  precision (theoretically, I don't know how precise the measurements actually are) would get to about 1 mm after the robot travelled around 10 km. It wouldn't actually overflow unless it drove beyond the edge of the galaxy.